syntax = "proto3";

package armonik.api.grpc.v1.sort_direction;

option csharp_namespace = "Armonik.Api.Grpc.V1.SortDirection";

/**
  * Represents every available sort directions.
  */
enum SortDirection {
  SORT_DIRECTION_UNSPECIFIED = 0; /** Unspecified. Do not use. */
  SORT_DIRECTION_ASC = 1; /** Ascending. */
  SORT_DIRECTION_DESC = 2; /** Descending. */
}
